### PR TITLE
docs: fix run local sp network

### PR DESCRIPTION
**Pull request by sysvm**
### Description

Fix run local sp network.

### Rationale

N/A

### CODE REVIEW

File: src/guide/storage-provider/run-book/run-testnet-SP-node.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ The following lists the recommended hardware requirements:
 * 1 Gbps network connection with upload/download speeds of 10MB/s+；
 * At least 1 TB disk space for backend storage;
 * 50GB+ SQL database;
-* Piece Store: AWS S3, MinIO(Beta)
-* 4 Greenfield Account with enough BNB tokens
+* Piece Store: AWS S3, MinIO(Beta);
+* 4 Greenfield accounts with enough BNB tokens.
```
Comment (will-2012) [inline src/guide/storage-provider/run-book/run-testnet-SP-node.md:18]: What about the `4`?

Comment (Fynnss) [reply src/guide/storage-provider/run-book/run-testnet-SP-node.md:18]: After adding the gc account, we will need five accounts here.

**Review by RenRick — APPROVED** [review]
LGTM

File: src/guide/storage-provider/run-book/run-testnet-SP-node.md (hunk 1 above)
Comment (Fynnss) [inline src/guide/storage-provider/run-book/run-testnet-SP-node.md:18]:
```suggestion
* 5 Greenfield accounts with enough BNB tokens.
```

File: src/guide/storage-provider/run-book/run-testnet-SP-node.md [hunk 2]
```diff
@@ -14,8 +14,8 @@ The following lists the recommended hardware requirements:
...
 
 ::: info
 Each storage provider will hold 4 different accounts serving different purposes:
```
Comment (Fynnss) [inline src/guide/storage-provider/run-book/run-testnet-SP-node.md:21]: Don't forget to add GC account here.

File: src/guide/storage-provider/run-book/run-testnet-SP-node.md (hunk 1 above)
Comment (sysvm) [reply src/guide/storage-provider/run-book/run-testnet-SP-node.md:18]: fixed

File: src/guide/storage-provider/run-book/run-testnet-SP-node.md (hunk 2 above)
Comment (sysvm) [reply src/guide/storage-provider/run-book/run-testnet-SP-node.md:21]: fixed

**Pull request merged**